### PR TITLE
Features grades info

### PR DESCRIPTION
**Pull request by Siupahc**
# Features

- You can now get infos about the grade by clicking on the grade the coach gives you. This allows user unfamiliar with the S A B C D tier system to understand their note.
- The AlertDialogue that explain how to use the MLCoach now appears only the first time

### CODE REVIEW

File: app/src/androidTest/java/com/android/sample/screen/CoachFeedBackScreenTest.kt [hunk 1]
```diff
@@ -49,7 +50,7 @@ class CoachFeedBackScreenTest {
     composeTestRule.onNodeWithTag("animatedText").performScrollTo().assertIsDisplayed()
     composeTestRule.onNodeWithTag("rankCircle").performScrollTo().assertIsDisplayed()
     composeTestRule
-        .onNodeWithTag("rankText")
+        .onNodeWithTag("rankText", useUnmergedTree = true)
```
Comment (nadaedh) [inline app/src/androidTest/java/com/android/sample/screen/CoachFeedBackScreenTest.kt:53]: Whats the point of using useUnmergedTree here ?

File: app/src/main/java/com/android/sample/ui/mlFeedback/CoachFeedbackScreen.kt [hunk 2]
```diff
@@ -131,6 +144,33 @@ fun CoachFeedbackScreen(navigationActions: NavigationActions, cameraViewModel: C
...
+        Text(
+            "The grades are in the following order: \n" +
+                "S\n" +
+                "A\n" +
+                "B\n" +
+                "C\n" +
+                "D\n" +
+                "S being the best and meaning you're doing it perfectly\n" +
+                "With D however, you should consider learning to do the exercise properly in order to train efficiently and not get injured")
+      }
```
Comment (nadaedh) [inline app/src/main/java/com/android/sample/ui/mlFeedback/CoachFeedbackScreen.kt:169]: Wouldn't it be better to add this text to the strings.xml file ?

**Review by nadaedh — COMMENTED** [review]
Once again a clear and concise PR, I added two comments for you to check out but great PR overall

File: app/src/androidTest/java/com/android/sample/screen/CoachFeedBackScreenTest.kt (hunk 1 above)
Comment (Siupahc) [reply app/src/androidTest/java/com/android/sample/screen/CoachFeedBackScreenTest.kt:53]: Tests wouldn't pass without it, but it did before, I'm not sure why but this fixed the issue

File: app/src/main/java/com/android/sample/ui/mlFeedback/CoachFeedbackScreen.kt (hunk 2 above)
Comment (Siupahc) [reply app/src/main/java/com/android/sample/ui/mlFeedback/CoachFeedbackScreen.kt]: I added it now

**Review by nadaedh — APPROVED** [review]
Nice work

**Pull request merged**